src/lib/run-dev.js

```diff
@@ -495,8 +497,7 @@ async function serveWebAction (req, res, actionConfig, distFolder, actionLoader
     distFolder,
     contextActionLoader: actionLoader
   }
-
-  if (invoker && !sequence) {
+  if (invoker && !sequence && !actionConfig.strictMode) {
```

Comment:
Where is the serveNonWebAction code? That needs to be resurrected here (it was deleted in the previous [PR](https://github.com/adobe/aio-cli-plugin-app-dev/pull/125)) and enabled by `--strict`. If not the e2e tests will fail since they expect 401s for non-web actions being called.